techniques/html/H44.html

```diff
@@ -231,75 +204,68 @@ <h2>Resources</h2>
                
             </ul>
          </section>
+         <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注:</div><div><p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。上記は、HTML 5.2§4.10.4. The label element を代わりに参照できる。</p></div></div>
```

Comment:
HTML Standardへのリンクを追加します。
```suggestion
         <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注:</div><div><p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。上記は、<a href="https://html.spec.whatwg.org/multipage/forms.html#the-label-element">HTML §4.10.4 The label element</a> を代わりに参照できる。</p></div></div>
```